src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.iOS/AppleCertificatePal.cs

```diff
@@ -0,0 +1,572 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Formats.Asn1;
+using System.Security.Cryptography;
+using System.Security.Cryptography.Apple;
+using System.Security.Cryptography.X509Certificates;
+using System.Security.Cryptography.Asn1;
+using System.Security.Cryptography.Asn1.Pkcs12;
+using System.Security.Cryptography.Asn1.Pkcs7;
+using System.Text;
+using System.Threading;
+using Microsoft.Win32.SafeHandles;
+
+namespace Internal.Cryptography.Pal
+{
+    internal sealed partial class AppleCertificatePal : ICertificatePal
```

Comment:
I'm having trouble seeing what's different in this version and the macOS version.  Can we get the parts that are the same in one shared partial and the parts that are different in more specific partials?  (I feel like it's just the loader routines?)

Reply:
> Can we get the parts that are the same in one shared partial and the parts that are different in more specific partials?

Sure. I wasn't sure how much would be shareable when I started and this file ended up about half and half. Similarly for `X509Pal`...